### DISCUSSION TITLE
linenames when using grid

### DISCUSSION THREAD

**Discussion by chrispcode** [chakra-ui/panda#1892] [Q&A] [answered]
Does panda support linenames when using grid?

I am trying to achieve the following

```ts
import { css } from 'styled-system/css';

export const layout = css({
  display: 'grid',
  gridTemplateColumns: `
    [full-start]
      minmax(16px, 1fr)
        [breakout-start]
          minmax(0, 16px)
            [content-start]
              minmax(min-content, 1024px)
            [content-end]
          minmax(0, 16px)
        [breakout-end]
      minmax(16px, 1fr)
    [full-end]
  `,
})
```

but I get: 
<img width="888" alt="image" src="https://github.com/chakra-ui/panda/assets/7029482/2adb2be1-d5a1-4ec6-9de5-1ba2ff7de16e">

I think it has something to do with how the classname is generated.
If I remove the linenames, "it works" (I rely on grid-column afterwords, so it doesn't really work)

**Comment by anubra266** [comment]
#1950

**Reply by anubra266** [reply, marked as answer]
@chrispcode This has now been released. Thanks for pointing out the issue

**Reply by chrispcode** [reply]
Thanks for the release, will test it out later today!

**Reply by chrispcode** [reply]
Tested it out, works like a charm! Fixed in version [0.26.0](https://github.com/chakra-ui/panda/releases/tag/%40pandacss%2Fshared%400.26.0), by [this PR](https://github.com/chakra-ui/panda/issues/1950)
